tests/test_launcher_remote.py

```diff
@@ -52,12 +52,12 @@ def test_launch_remote_instance(monkeypatch, new_solver_session):
     # Assert: PyFluent went through the pypim workflow
     assert mock_is_configured.called
     assert mock_connect.called
-    if os.getenv("FLUENT_IMAGE_TAG"):
-        product_version = docker_image_version.get_version_for_filepath()
-    else:
-        product_version = "".join(launcher.get_ansys_version().split("."))[:-1]
+    # if os.getenv("FLUENT_IMAGE_TAG"):
+    #     product_version = docker_image_version.get_version_for_filepath()
+    # else:
+    #     product_version = "".join(launcher.get_ansys_version().split("."))[:-1]
```

Comment:
Do you want to remove the commented lines>

Reply:
yes, Done.
Thank you.